spec/controllers/catalog_controller_spec.rb

```diff
@@ -974,6 +974,54 @@
       end
     end
 
+    describe '#fetch_ovf_template_details' do
+      let(:user)                     { FactoryBot.create(:user_with_group) }
+      let(:admin_user)               { FactoryBot.create(:user, :role => "super_administrator") }
+      let(:ems)                      { FactoryBot.create(:ems_vmware) }
+      let(:container_template)       { FactoryBot.create(:container_template, :ext_management_system => ems) }
+      let(:service_template_catalog) { FactoryBot.create(:service_template_catalog) }
+      let(:ovf_template)             { FactoryBot.create(:orchestration_template_vmware_infra, :ems_id => ems.id) }
+      let(:datacenter)               { FactoryBot.create(:vmware_datacenter) }
+      let(:resource_pool)            { FactoryBot.create(:resource_pool) }
+      let(:ems_folder)               { FactoryBot.create(:ems_folder) }
```

Comment:
Should the rest of these also belong to the `ems` you created above?  It might not be required but probably best practice